enatega-multivendor-web/src/components/HomeScreen/OrderFavorites/OrderFavorites.jsx

```
@@ -36,14 +38,15 @@ const OrderFavorites = () => {
             </Typography>
 
             <Box>
-              <a href="https://play.google.com/store/apps/details?id=com.enatega.multivendor" className={classes.stores}>
+              <Button href="https://play.google.com/store/apps/details?id=com.enatega.multivendor" 
+              className={classes.stores}>
               <img src={appStore} alt="playStore" > 
               </img>
```

Biome (1.9.4)

[error] img is a void element tag and must not have children.

Unsafe fix: Remove the children.


(lint/correctness/noVoidElementsWithChildren)